internal/rpc/server.go

```diff
@@ -258,6 +260,16 @@ func MustServeForever(params *ServerParams) {
 	serveUntilKilledFunc()
 }
 
+func instrumentHTTPHandler(handler http.Handler, params *ServerParams) http.Handler {
```

Comment:
null check for params?

Reply:
Not needed. Don't pass null like that and it's required for it to work.